_includes/header.html

```diff
@@ -23,3 +23,4 @@
         <li><a href="/contact-us">Contact Us</a></li>
     </ul>
 </nav>
+<script src="/js/scripts.js"></script>
```

Comment:
Normally you'll want to put any scrips at the bottom of the body/below the footer because they take a long time to download (and block all HTML rendering below until downloaded). Even though this script relates to the navbar, putting it at the bottom is a good idea. You can also put them in the `<head>` and use the `defer` attribute.